src/odemis/acq/fastem.py

```diff
@@ -900,6 +906,32 @@ def _create_acquisition_metadata(self):
         self._detector.updateMetadata({model.MD_EXTRA_SETTINGS: json.dumps(selected_settings)})
         return selected_settings
 
+    def _calculate_beam_shift_cor_indices(self, n_beam_shifts=10):
```

Comment:
I think this calculation can go to FastEMROA to reduce the overhead during acquisition

Reply:
It's pretty fast so I decided to put it here to reduce overhead during drawing, and then it does not have to be recalculated when a shape is resized. Let me know if you still think that I should move it to FastEMROA